src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  const accountanting = [];
+
+  const getSum = (x) => {
+    return function(y) {
```

Comment:
```suggestion
    return function(y) {
      accountanting++;
```
Increment that value after each function call